### PR TITLE
feat(settings/keybinds): redesign and connect to iridium

### PR DESCRIPTION
**Pull request by jasonwoodland**
**What this PR does** 📖
- Refactor keybind settings page
- Redesign UI
- Introduce reusable InteractablesKeybindInput component

**Which issue(s) this PR fixes** 🔨
AP-1992

**Special notes for reviewers** 🗒️

**Additional comments** 🎤

### CODE REVIEW

File: components/interactables/KeybindInput/KeybindInput.html [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    <div v-else class="key-list">
+      <template v-for="key, index in keys">
+        <kbd class="key">{{key}}</kbd>
+        <div v-if="index !== keys.length - 1" class="plus">+</div>
+      </template>
+    </div>
+    <div v-if="isListening" key="editing-controls" class="controls">
+      <InteractablesButton
+        type="dark"
+        v-tooltip.top="'Cancel'"
```
Comment (stavares843) [inline components/interactables/KeybindInput/KeybindInput.html:23]: can we add this to locales? 🔨

File: components/interactables/KeybindInput/KeybindInput.html [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+        v-tooltip.top="'Save'"
+        :action="save"
+        :disabled="!canSave"
+      >
+        <CheckIcon size="1x" />
+      </InteractablesButton>
+    </div>
+
+    <div v-else key="normal-controls" class="controls">
+      <InteractablesButton type="dark" v-tooltip.top="'Edit'" :action="edit">
```
Comment (stavares843) [inline components/interactables/KeybindInput/KeybindInput.html:40]: can we add this to locales? 🔨

File: locales/en-US.js [hunk 3]
```diff
@@ -422,10 +422,10 @@ export default {
...
         subtitle:
           'A list of the default keybinds for quickly navigating and using Satellite.',
-        mute: 'Toggle Mute',
-        settings: 'Open Settings',
-        deafen: 'Toggle Deafen',
-        call: 'Call Active Chat',
+        toggleMute: 'Toggle Mute',
+        openSettings: 'Open Settings',
+        toggleDeafen: 'Toggle Deafen',
+        callActiveChat: 'Call Active Chat',
```
[the remark this answers is not in the document]
Comment (jasonwoodland) [reply locales/en-US.js:428]: I had to do this because we're iterating over object keys to programatically generate the locale string.

**Comment by molimauro** [comment]
The PR looks great! I tried with the space char and the empty indicator is weird, can we put something like the `␣` ? Also are we following a figma design from the Ui/Ux team?

Last: the fact that i cannot use more than one letter in a keybing is a library constraint or our choice? :)

**Review by aewing — APPROVED** [review]

**Pull request merged**